packages/ui-components/src/components/DataTable/useDataTableExport.js

```diff
@@ -28,9 +28,18 @@ export const useDataTableExport = (columns, data, title, startDate, endDate) =>
         ? tableData.map(row =>
             tableColumns.map(col => {
               const value = row.values[col.id];
-              const valueIsPercentageString = value.includes('%');
-              const num = valueIsPercentageString ? value : parseFloat(value);
-              // if it's a number, return in number format
+
+              // Attempt to convert to a moment object if is a string that can't be converted to a number (isNaN). If it's a date string, return in date format, otherwise continue
+              // This check will handle percentage strings as well as date strings, and any other non-numeric strings
+              if (typeof value === 'string' && isNaN(value)) {
+                const momentObj = displayStringToMoment(value);
+                // if it's a date string, return in date format
+                if (momentObj.isValid()) return momentObj.format('DD/MM/YYYY');
+                return value;
```

Comment:
I think this will break regular number fields

`displayStringToMoment` attempts to parse the string through a number of standard formats, but that includes things like `YYYY`, so if I enter _any_ four digit number it'll get parsed and then turned into `01/01/1234`?

(I don't recall if `YYYY` will successfully parse shorter numbers, but I think it does. So the number `1` get turned into `01/01/0001`)

Reply:
Ordinarily yes it probably would break it, but because it's inside this check: `if (typeof value === 'string' && isNaN(value))` normal numbers will never get there. It can only be strings that aren't immediately recognisable as a number, e.g. `11/2/10`

Reply:
Oh yup! My brain glossed over the `isNaN`, that makes sense